packages/components/__tests__/Chip.spec.jsx

```diff
@@ -0,0 +1,301 @@
+import { fireEvent, render, screen } from '@testing-library/react'
+import { expect, vi } from 'vitest'
```

Comment:
```suggestion
import { vi } from 'vitest'
```
And i am not sure but maybe you can delete `vi` too, because we have it in `setup-tests.ts`

Reply:
I check it and yes, you can delete this line. `vi` is not need to import

Reply:
@Iarynovskyi i've removed it.